app/controllers/bus_stops_controller.rb

```diff
@@ -64,6 +64,7 @@ def outdated
 
   def update
     @stop.assign_attributes stop_params
+    @stop.assign_completed_by current_user
```

Comment:
Style comment, slightly misleading in my opinion  to say "assign completed by current user" when the majority of the time, we don't actually do what those words suggest.